doofinder/includes/class-post.php

```diff
@@ -347,8 +347,16 @@ public function format_for_api() {
 	 * @return string[]
 	 */
 	private function get_categories() {
-		$all_categories       = get_categories();
-		$post_categories      = wp_get_post_categories( $this->post->ID, array( 'fields' => 'all' ) );
+		$all_categories = get_categories();
+		$try_portfolio_cat = get_terms('portfolio_category');
+		$all_portf_categories = (is_wp_error($try_portfolio_cat) ? [] : $try_portfolio_cat);

```

Comment:
Don't you think it would be better adding a conditional to detect if the taxonomy exists instead of checking if the get_terms returns an error?
```suggestion
		$portfolio_category_enabled = taxonomy_exists("portfolio_category)
		if($portfolio_category_enabled){
			$all_categories = array_merge($all_categories,  get_terms('portfolio_category'));
		}
```